src/encoded/types/experiment.py

```diff
@@ -184,8 +184,7 @@ def biosample_summary(self,
                                             tag_dict = {'location': tag['location']}
                                             if tag.get('promoter_used'):
                                                 tag_dict['promoter'] = request.embed(
-                                                    tag.get('promoter_used'),
-                                                            '@@object').get['label']
+                                                    tag.get('promoter_used'), '@@object').get['label']
```

Comment:
We shouldn't make changes not related to the feature.

Reply:
ok